<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackCreate</class>
 <widget class="QDialog" name="PackCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create pack window </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu{
background-color:   rgb(50, 50, 50);
color: white;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
}
QMenu::item:disabled{
background-color: rgb(147, 147, 147);
color: rgb(90, 90, 90);
}

QMenu::item:disabled:selected{
background-color:  rgb(147, 147, 147);
}



QMenu::item:selected  {
background-color: rgb(198, 66, 0);
}

QMenu::separator {
height: 0px;
}


#PackCreate{
background-color: rgb(50, 50, 50);
}

QLabel{
text-align: center;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
}

QPushButton{
background-color: rgb(80, 80, 80);
border-radius: 8px;
padding:8px  5px;
text-align: center;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
}

QPushButton:hover{
background-color: rgb(198, 66, 0);
}

QLineEdit{
background-color: rgb(51, 51, 51);
border:none;
border-bottom: 2px solid  rgba(198, 66, 0,100);
font-family: &quot;Georgia&quot;;
font-size: 14px;
color: rgb(232, 232, 232);
}

QSpinBox{
border: 1px solid rgba(198, 66, 0,140);
background-color: rgb(50, 50, 50);
border-radius: 4px;
padding: 1px;
color:white;
}

QSpinBox::down-arrow{
image: url(:/Icons/Icons/ComboBoxArrowDown.svg);
width:14px;
height:14px;
}

QSpinBox::down-button {
width: 14px;
border-width: 1px;
border-top-width: 0;
}

QSpinBox::up-button {
width: 14px;
border-width: 1px;
border-top-width: 0;
}

QSpinBox::up-arrow{
image: url(:/Icons/Icons/ComboBoxArrowDownOn.svg);
width:14px;
height:14px;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Accuracy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ApproximateLine">
           <property name="maxLength">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Number of unknowns</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinUknownBox">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QFormLayout" name="EquationLayout"/>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <layout class="QFormLayout" name="ApproximationLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="AddButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Append</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="BackButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="ExitButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
